### DISCUSSION TITLE
Float label with p:autoComplete and multiple="true" not working

### DISCUSSION THREAD

**Discussion by Cprusinowski** [primefaces/community#1987] [PrimeFaces] [answered]
Hi,

When I try to use code below then float label doesn't work.
Cause of this is multiple="true".
Does anyone have a solution?

```
<span class="ui-float-label">
    <p:autoComplete multiple="true" />
    <p:outputLabel for="@previous" value="AutoComplete" />
</span>
```

**Comment by melloware** [comment]
Here is a runnable reproducer showing the issue: 
[discussion-1987.zip](https://github.com/primefaces/community/files/15389742/discussion-1987.zip)

**Reply by melloware** [reply, marked as answer]
@Cprusinowski I converted your discussion into an issue on PF Github: https://github.com/primefaces/primefaces/issues/11974

**Reply by melloware** [reply]
This is a tricky one because in Multiple mode the AutoComplete is rendered with a different DOM structure than a normal input.

**Reply by melloware** [reply]
PR submitted: https://github.com/primefaces/primefaces/pull/13108
